python/tests/integration/arcticdb/version_store/test_snapshot.py

```diff
@@ -491,4 +498,349 @@ def assert_0_delete_marker(lib, storage):
     assert_0_delete_marker(lib, storage)
 
     lib.remove_from_snapshot("snap", ["s2"], [s2_ver])
-    assert_0_delete_marker(lib, storage)
\ No newline at end of file
+    assert_0_delete_marker(lib, storage)
+
+
+def test_delete_snapshot_basic_flow(basic_store):
+    lib = basic_store
+    symbol = "sym1"
+    snap1 = "snap1"
+    snap2 = "snap2"
+
+    df_0 = create_data_frame(10, 0, 10)
+    df_1 = create_data_frame(10, 10, 20)
+    df_combined = pd.concat([df_0, df_1])
+
+    lib.write(symbol, df_0)
+    lib.snapshot(snap1)
+    lib.append(symbol, df_1)
+    lib.snapshot(snap2)
+    versions_initial = [v["version"] for v in lib.list_versions()]
+
+    assert sorted(lib.list_snapshots()) == [snap1, snap2] 
+    assert df_combined.equals(lib.read(symbol).data)
```

Comment:
In most places around our codebase we use `assert_frame_equal` which I believe is more strict than `df.equals`. What do you think about using this here as well?